Le test que j'ai écrit fail, mais je ne sais pas pourquoi...
J'ai l'output : 
```
(zdsenv) [nils@pineapple:zestedesavoir/zds-site][1]$ ./manage.py test zds.notification.tests.tests_tricky.SubscriptionsTest
Creating test database for alias 'default'...
 DEBUG  zds.utils.templatetags.emarkdown Result , {'disableToc': True, 'languages': [], 'depth': -1}, []
DEBUG:zds.utils.templatetags.emarkdown:Result , {'disableToc': True, 'languages': [], 'depth': -1}, []
 DEBUG  zds.utils.templatetags.emarkdown Result , {'disableToc': True, 'languages': [], 'depth': -1}, []
DEBUG:zds.utils.templatetags.emarkdown:Result , {'disableToc': True, 'languages': [], 'depth': -1}, []
F
======================================================================
FAIL: test_no_emails_for_those_who_have_none (zds.notification.tests.tests_tricky.SubscriptionsTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/nils/Developpement/zestedesavoir/zds-site/zds/notification/tests/tests_tricky.py", line 402, in test_no_emails_for_those_who_have_none
    self.assertEqual(1, len(mail.outbox))
AssertionError: 1 != 0

----------------------------------------------------------------------
Ran 1 test in 0.818s

FAILED (failures=1)
Destroying test database for alias 'default'...
(zdsenv) [nils@pineapple:zestedesavoir/zds-site][1]$
```

@artragis est-ce que tu sais ce qui peut faire que le mail n'est pas envoyé à l'utilisateur normal ?

--- a/zds/notification/models.py
+++ b/zds/notification/models.py
@@ -88,7 +88,7 @@ class Subscription(models.Model):
 
         receiver = self.user
 
-        # This can happen when an user subscribes via social networks without providing an e-mail adress
+        # This can happen when a user subscribes via social networks without providing an e-mail address
         if not receiver.email:
             return
 

